### PR TITLE
 docs(project): adding project specific files

### PR DESCRIPTION
**Pull request by mynktl**
Changes:
- This PR adds project specific files to guide contributors
    List of the files is as below: 
     - [.github/ISSUE_TEMPLATE/bug_report.md](https://github.com/openebs/libcstor/blob/8349497e85e62dc9f5f445deec1eb36436b9f397/.github/ISSUE_TEMPLATE/bug_report.md)
     - [.github/ISSUE_TEMPLATE/feature-request.md](https://github.com/openebs/libcstor/blob/8349497e85e62dc9f5f445deec1eb36436b9f397/.github/ISSUE_TEMPLATE/feature-request.md)
     - [.github/pull_request_template.md](https://github.com/openebs/libcstor/blob/8349497e85e62dc9f5f445deec1eb36436b9f397/.github/pull_request_template.md)
     - [CONTRIBUTING.md](https://github.com/openebs/libcstor/blob/8349497e85e62dc9f5f445deec1eb36436b9f397/CONTRIBUTING.md)
     - [GOVERNANCE.md](https://github.com/openebs/libcstor/blob/8349497e85e62dc9f5f445deec1eb36436b9f397/GOVERNANCE.md)
     - [RELEASE.md](https://github.com/openebs/libcstor/blob/8349497e85e62dc9f5f445deec1eb36436b9f397/RELEASE.md)
     - [code-standard.md](https://github.com/openebs/libcstor/blob/8349497e85e62dc9f5f445deec1eb36436b9f397/code-standard.md)
     - [developer-setup.md](https://github.com/openebs/libcstor/blob/8349497e85e62dc9f5f445deec1eb36436b9f397/developer-setup.md)

Signed-off-by: mayank <mayank.patel@mayadata.io>

### CODE REVIEW

**Review by vishnuitta — APPROVED** [review]
changes are good

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+**What did you expect to happen:**
+
+
+**The output of the following commands and cstor pod logs will help us better understand what's going on**:
+(Pasting long output into a [GitHub gist](https://gist.github.com) or other [Pastebin](https://pastebin.com/) is fine.)
+
+* `kubectl logs deployment/maya-apiserver -n openebs`
+* `kubectl logs deployment/<pv-name>-target -n <openebs_namespace> -c cstor-istgt`
+* `kubectl get pods -n openebs`
```
Comment (kmova) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]: should we include kubectl get of the custom resources like cvr, cv and csp here?

Comment (mynktl) [reply .github/ISSUE_TEMPLATE/bug_report.md:20]: done

**Comment by mynktl** [comment]
@kmova I've address the review comment. PTAL.

**Review by kmova — APPROVED** [review]

**Pull request merged**